shared/js/intl/l20n.js

```diff
@@ -1837,8 +1837,14 @@
 
       const pseudoentries = Object.create(null);
       for (let key in entries) {
-        pseudoentries[key] = walkEntry(
-          entries[key], pseudo[lang.code].process);
+        if (['datePickerOrder',
+            'timePickerOrder',
+            'durationPattern'].includes(key)) {
```

Comment:
`includes` doesn't transpile well to es5 without a special babel plugin.  I'd suggest using `indexOf`.